arch/arm/boot/dts/imx6ul-ts7100-5.dtsi

```diff
@@ -0,0 +1,171 @@
+		compatible = "microchip,wilc3000";
+		reg = <0>;
+		spi-max-frequency = <20000000>;
+		reset-gpios = <&gpio7 7 GPIO_ACTIVE_HIGH>;
+		chip_en-gpios = <&gpio1 14 GPIO_ACTIVE_HIGH>;
+		interrupt-parent = <&gpio1>;
+		interrupts = <11 GPIO_ACTIVE_HIGH>;
+	};
+};
+
+&gpio1 {
+	gpio-line-names =
+		"RS485_5_TXEN",  "RTC_BATT_FAIL", "I2C_1_CLK",     "I2C_1_DAT",
+		"AN_CH0",        "AN_CH1",        "",              "",
+		"",              "AN_CH2",        "RS485_2_TXEN",  "",
+		"",              "",              "",              "",
+		"",              "",              "",              "",
+		"",              "",              "",              "",
+		"",              "",              "",              "",
+		"",              "",              "",              "";
+};
+
+&iomuxc {
+	pinctrl-names = "default";
+	pinctrl-0 = <&ts7100_5_pinctrl_hog>;
+
+	imx6ul-ts7100-5 {
+		ts7100_5_pinctrl_hog: hoggrp {
+			fsl,pins = <
+				/*
+				 * All GPIO should be 0x1b020 unless special
+				 * 0x1b020 == Hyst., 100k PU, 50 mA drive
+				 * 0x1a020 == no pull resistor
+				 * 0x13020 == 100k PD
+				 */
+
+				MX6UL_PAD_JTAG_TDO__GPIO1_IO12		0x1b020
+				MX6UL_PAD_SNVS_TAMPER6__GPIO5_IO06	0x1b020
+				MX6UL_PAD_SNVS_TAMPER7__GPIO5_IO07	0x1b020
+				MX6UL_PAD_SNVS_TAMPER8__GPIO5_IO08	0x1b020
+				MX6UL_PAD_SNVS_TAMPER9__GPIO5_IO09	0x1b020
+
+				/* POWER_FAIL */
+				MX6UL_PAD_SNVS_TAMPER0__GPIO5_IO00	0x1b020
+			>;
+		};
+
+		pinctrl_adc1: adc1grp {
+			fsl,pins = <
+				MX6UL_PAD_GPIO1_IO01__GPIO1_IO01	0x1a020
+				MX6UL_PAD_GPIO1_IO04__GPIO1_IO04	0x1a020
+				MX6UL_PAD_GPIO1_IO05__GPIO1_IO05	0x1a020
+				MX6UL_PAD_GPIO1_IO08__GPIO1_IO08	0x1a020
+			>;
+		};
+
+		pinctrl_ecspi3: ecspi3grp {
+			fsl,pins = <
+				MX6UL_PAD_NAND_CE0_B__ECSPI3_SCLK	0x1b020
+				MX6UL_PAD_NAND_CE1_B__ECSPI3_MOSI	0x1b020
+				MX6UL_PAD_NAND_CLE__ECSPI3_MISO		0x1b020
+				MX6UL_PAD_NAND_READY_B__GPIO4_IO12	0x1b020
+				MX6UL_PAD_JTAG_TMS__GPIO1_IO11		0x1b020
+				MX6UL_PAD_JTAG_TCK__GPIO1_IO14		0x1b020
+			>;
+		};
+
+		pinctrl_uart2: uart2grp {
+			fsl,pins = <
+				MX6UL_PAD_UART2_TX_DATA__UART2_DCE_TX	0x1b0b1
+				MX6UL_PAD_UART2_RX_DATA__UART2_DCE_RX	0x1b0b1
+				MX6UL_PAD_GPIO1_IO09__GPIO1_IO09	0x13020
+			>;
+		};
+		pinctrl_uart3: uart3grp {
+			fsl,pins = <
+				MX6UL_PAD_UART3_RX_DATA__UART3_DCE_RX	0x1b0b1
+				MX6UL_PAD_UART3_TX_DATA__UART3_DCE_TX	0x1b0b1
+				MX6UL_PAD_UART3_CTS_B__UART3_DCE_CTS	0x1b0b1
+				MX6UL_PAD_UART3_RTS_B__UART3_DCE_RTS	0x1b0b1
+			>;
+		};
+
+		pinctrl_uart4: uart4grp {
+			fsl,pins = <
+				MX6UL_PAD_UART4_RX_DATA__UART4_DCE_RX	0x1b0b1
+				MX6UL_PAD_UART4_TX_DATA__UART4_DCE_TX	0x1b0b1
+			>;
+		};
+
+		pinctrl_uart5: uart5grp {
+			fsl,pins = <
+				MX6UL_PAD_UART5_RX_DATA__UART5_DCE_RX	0x1b0b1
+				MX6UL_PAD_UART5_TX_DATA__UART5_DCE_TX	0x1b0b1
+				MX6UL_PAD_GPIO1_IO00__GPIO1_IO00	0x13020
+			>;
+		};
+	};
+};
+
+&uart2 {
+	uart-has-rtscts;
+	rts-gpios = <&gpio1 9 GPIO_ACTIVE_HIGH>;
+	linux,rs485-enabled-at-boot-time;
+	dma-names = "", "";
+};
```

Comment:
Missing pinctrl parameters?

Reply:
Good catch... that might be why I couldn't make that rts gpio work.